src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="productcard" data-qa="card">
+      <div class="images">
+        <img
+          class="images_item"
+          src="images/imac.jpeg"
+         alt="foto"
+     >
+      </div>
+      <div class="description">
+        <div class="description__name">
+        APPLE A1419 iMac 27" Retina 5K
+        Monoblock (MNED2UA/A)
+        </div>
+        <div class="description_code">
+          Product code: 195434
+        </div>
+        <div class="description__rating">
+          <div class="stars">
+            <div class="star"></div>
+            <div class="star"></div>
+            <div class="star"></div>
+            <div class="star"></div>
+            <div class="star"></div>
+          </div>
+          <div class="reviews">
+            Reviews: 5
+          </div>
+        </div>
+        <div class="price">
+          <div class="title">
+            Price:
+          </div>
+          <div class="value">
+            $2,199
+          </div>
+
+        </div>
+        <div class="link" data-qa="hover">
+          <a  class="button" href="%">BUY</a>
+        </div>
+      </div>
+
+    </div>
+  </body>
```

Comment:
no space between parent and child element
```suggestion
        </div>
      </div>
    </div>
  </body>
```

Reply:
is not fixed

Reply:
is still not fixed